server/routers/treesRouter.ts

```diff
@@ -40,9 +40,10 @@ router.get(
   '/:id',
   handlerWrapper(async (req, res) => {
     Joi.assert(req.params.id, Joi.number().required());
+    Joi.assert(req.query.organization_id, Joi.number());
     const repo = new TreeRepository(new Session());
-    const exe = TreeModel.getById(repo);
-    const result = await exe(req.params.id);
+    const exe = TreeModel.getByIdOrOrganizationId(repo);
+    const result = await exe(req.params.id, req.query.organization_id);
```

Comment:
@mayankbucha we have some misunderstanding on this problem (#173) what I mean is that we need to fix the route: `GET /trees` (rather than `GET /trees/id` ) for the `GET /trees` (https://dev-k8s.treetracker.org/query/trees?organization_id=178) return body is: 

```
{
  "total": null,
  "offset": 0,
  "limit": 20,
  "trees": [
    {
      "id": 178,
      "time_created": "2021-02-04T01:49:24.000Z",
      "time_updated": "2021-02-04T01:49:24.000Z",
      "missing": false,
      "priority": false,
      "cause_of_death_id": null,
      "planter_id": 323,
      "primary_location_id": null,
      "settings_id": null,
      "override_settings_id": null,
```

Here, the `id:178` is wrong, which should be the tree id, but we wrongly put organization id here (178)

Reply:
Ok, I will make a new PR and address this in that.